data-processing/perl/setup.sh

```diff
@@ -0,0 +1,9 @@
+# Fetch sources for AI2 clone of LaTeXML.
```

Comment:
does `data-processing/Dockerfile` need to get updated to call this script?

Reply:
It sure does! Great catch.